raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -246,7 +246,9 @@ public KafkaRaftClient(
             logContext,
             random);
         this.kafkaRaftMetrics = new KafkaRaftMetrics(metrics, "raft", quorum);
-        kafkaRaftMetrics.updateNumUnknownVoterConnections(quorum.remoteVoters().size());
+        // All Raft voters are statically configured and known at startup
+        // so there are no unknown voter connections. Report this metric as 0.
+        kafkaRaftMetrics.updateNumUnknownVoterConnections(0);
```

Comment:
Perhaps we should file a jira to follow up with removal of this metric if it is not reporting anything useful.

Reply:
Yeah! Was planning on filing a JIRA and KIP for its removal. Will do that next week. :)